### PR TITLE
Support force install

### PR DESCRIPTION
**Pull request by anisimkov**

### CODE REVIEW

File: langkit/libmanage.py [hunk 1]
```diff
@@ -257,6 +257,11 @@ def internal(*args, **kwargs):
             help='Installation directory.'
         )
 
+        install_parser.add_argument(
+            '--force', '-f', action='store_true', dest='force', default=False,
```
Comment (pmderodat) [inline langkit/libmanage.py]: Both `dest` and `default` arguments can be removed (`default=False` is implied by `action='store_true'` and `dest='force'` is implied by `--force`).

Comment (anisimkov) [reply langkit/libmanage.py]: ok, good, i'll fix

Comment (anisimkov) [reply langkit/libmanage.py]: Fixed

Comment (pmderodat) [reply langkit/libmanage.py]: Thanks!

**Pull request merged**